src/trunk.c

```diff
@@ -8182,6 +8197,62 @@ trunk_print(platform_log_handle *log_handle, trunk_handle *spl)
    trunk_print_subtree(log_handle, spl, spl->root_addr);
 }
 
+/*
+ * Print meta-page's linked list for one routing filter at address 'meta_head'.
+ */
+void
+trunk_print_filter_metapage_list(platform_log_handle *log_handle,
+                                 trunk_handle        *spl,
+                                 uint64               meta_head)
+{
+   platform_log(log_handle,
+                "\nFilter Metadata page starting from meta_head=%lu\n{\n",
+                meta_head);
+   mini_unkeyed_print(spl->cc, meta_head, PAGE_TYPE_FILTER);
```

Comment:
Here is a sample output from this call, for a routing filter off of trunk's root node:

```
Filter Metadata page starting from meta_head=6200360960
{
---------------------------------------------
| Mini Allocator -- meta_head=6200360960 (pgnum=1513760)
|-------------------------------------------|
{
|-------------------------------------------|
| meta addr=6200360960 (pgnum=1513760), num_entries=4
| next_meta_addr=0 (pgnum=0), pos=52
|-------------------------------------------|
| idx |        extent_addr |     extent_num |
|-------------------------------------------|
|   0 |         6200492032 |          47306 |
|   1 |         6200623104 |          47307 |
|   2 |         6200754176 |          47308 |
|   3 |         6200885248 |          47309 |
|-------------------------------------------|
}

Found 1 meta-data pages tracking 4 extents.

}
```